tests/integration/components/frost-select-test.js

```diff
@@ -412,6 +412,45 @@ describeComponent(...integration('frost-select'), function () {
         expect(onChange.callCount, 'OnChange is not called').to.equal(0)
         expect(onFocus.callCount, 'onFocus is not called').to.equal(0)
       })
+
+      describe('when input is disabled and then re-enabled', function () {
+        beforeEach(function () {
+          this.set('disabled', true)
+          this.set('disabled', false)
```

Comment:
Doing two set calls back to back like this doesn't read well and doesn't guarantee they happen in different run loops. Instead can you do:

```js
describe('when disabled', function () {
  beforeEach(function () {
    this.set('disabled', true)
  })

  describe('when re-enabled', function () {
    …
  })

  describe('when enabled and tab index changed', function () {
    …
  })
})

```